packages/terra-menu/src/MenuDivider.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
This is component would be used by consumers when the menu items need to be separated into sections. I had considered making a Menu.Section component for this, but then there would be css logic needed to hide the top divider if the section is at the beginning of the list, hide the bottom divider if the section is at the end, and hide one if there are two sections next to each other. And that also complicates the height logic.